src/main/java/org/assertj/core/api/AbstractLocalDateAssert.java

```diff
@@ -264,6 +267,26 @@ public SELF isNotEqualTo(String localDateAsString) {
     return isNotEqualTo(parse(localDateAsString));
   }
 
+  /**
+   * Same assertion as {@link #isEqualTo(Object)} (where Object is expected to be {@link javax.xml.datatype.XMLGregorianCalendar})
+   * but here you pass {@link javax.xml.datatype.XMLGregorianCalendar}
+   * <p>
+   * Example :
+   * <pre><code class='java'> // use XMLGregorianCalendar  as-is without conversion
+   * assertThat(parse("2000-01-01")).isEqualTo(xmlGregorianCalendar);</code></pre>
+   *
+   * @param xmlGregorianCalendar to compare with actual {@link LocalDate}.
+   * @return this assertion object.
+   * @throws AssertionError if the actual {@code LocalDate} is {@code null}.
+   * @throws IllegalArgumentException if given calendar is null.
+   * @throws AssertionError if the actual {@code LocalDate} is not equal to the {@link XMLGregorianCalendar}
+   *  year, month and day. (Time and Timezone are ignored).
+   */
+  public SELF isEqualTo(XMLGregorianCalendar xmlGregorianCalendar) {
```

Comment:
It should be possible to move this method (or at least its implementation) in `AbstractTemporalAssert` but still override the method in order to write the proper javadoc.